examples/standalone/runfile/dynamics.py

```diff
@@ -1,120 +1,152 @@
+import json
+import pathlib
 from argparse import ArgumentParser
+from datetime import datetime
 
-import mpi4py
+import git
+import numpy as np
 import serialbox
 import yaml
+from mpi4py import MPI
 
 import fv3core
 import fv3core._config as spec
 import fv3core.stencils.fv_dynamics as fv_dynamics
 import fv3core.testing
 import fv3gfs.util as util
-from fv3core.testing import GlobalTimer, write_to_json
+from fv3gfs.util import Timer
+
+
+def print_and_write_global_timings(
+    timer, experiment_name, time_step, backend, comm, root=0
+):
+    is_root = comm.Get_rank() == root
+    recvbuf = np.array(0.0)
+    experiment = {}
+    now = datetime.now()
+    sha = git.Repo(
+        pathlib.Path(__file__).parent.absolute(), search_parent_directories=True
+    ).head.object.hexsha
+    dt_string = now.strftime("%d/%m/%Y %H:%M:%S")
+    filename = now.strftime("%Y-%m-%d-%H-%M-%S")
+    experiment["setup"] = {}
+    experiment["setup"]["experiment time"] = dt_string
+    experiment["setup"]["data set"] = experiment_name
+    experiment["setup"]["timesteps"] = time_step
+    experiment["setup"]["hash"] = sha
+    experiment["setup"]["version"] = "python/" + backend
+    experiment["times"] = {}
+    for name, value in timer.times.items():
+        if is_root:
```

Comment:
can we move this down to the second `is_root` block (Line 49) to only have one?

Reply:
I don't know if we can do that, since the other `if is_root` block is inside a loop over label/operation. Is it better to take it out of the if-statement and just have every rank set `experiment["times"][name] = {}` or to have the if-statement there?

Reply:
you're absolutely right